using-mobile-extensions/adobe-target-vec.md

```diff
@@ -0,0 +1,144 @@
+3. The default configuration options loads Target VEC activities as a blocking call on App launch. See [Implementation Methods for Target VEC](implementation-methods-for-target-vec) for details.
+4. Click **Save**.
+5. Follow the publishing process to update SDK configuration
+
+## Add Target VEC to your app
+
+#### Java
+
+1. Add the Target VEC extension to your project using the app's Gradle file.
+2. Import the Target VEC extension in your application's main activity.  `import com.adobe.marketing.mobile.*;`
+3. Add the Target VEC library to your project via your `Podfile` by adding `pod 'ACPTargetVEC'`
+
+#### Objective-C
+
+Import the Target and Identity library.
+
+```objectivec
+   #import "ACPCore.h"
+   #import "ACPTargetVEC.h"
+   #import "ACPTarget.h"
+   #import "ACPIdentity.h"
+   #import "ACPTargetRequestObject.h"
+   #import "ACPTargetPrefetchObject.h"
+```
+
+#### Swift
+
+```swift
+   #import ACPCore
+   #import ACPTarget
+   #import ACPTargetVEC
+   #import ACPIdentity
+```
+
+### Register Target VEC with Mobile Core
+
+{% tabs %}
+{% tab title="Android" %}
+#### Java
+
+After calling the `setApplication()` method in the `onCreate()` method, register Target VEC with Mobile Core.
+
+Here is code sample that calls these set up methods:
+
+```java
+public class TargetApp extends Application {
+
+ @Override
+ public void onCreate() {
+     super.onCreate();
+     MobileCore.setApplication(this);
+     try {
+         //Other Extensions that you need
+         TargetVEC.registerExtension();
+         Target.registerExtension();
+     } catch (Exception e) {
+         //Log the exception
+     }
+ }
+}
+```
+{% endtab %}
+
+{% tab title="iOS" %}
+#### Objective-C
+
+1. In your app's `didFinishLaunchingWithOptions` function register the Target VEC extension
+
+```objectivec
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+  //Other Extensions that you need
+  [ACPTargetVEC registerExtension];
+  [ACPTarget registerExtension];
+
+  // Override point for customization after application launch.
+  return YES;
+}
+```
+
+#### Swift
+1. In your app's `didFinishLaunchingWithOptions` function register the Target VEC extension
+
+```swift
+func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+  //Other Extensions that you need
+  ACPTargetVEC.registerExtension()
+  ACPTarget.registerExtension()
+  
+  return true
+}
+```
+{% endtab %}
+{% endtabs %}
+
+## Implementation Methods for Target VEC   <a id="implementation-methods-for-target-vec"></a>
+The Target VEC extension retrieves the relevant Target experiences for your app through a single network request. Offers are retrieved via this network call and applied automatically on the targeted screens. It should be noted that no network requests are made to retrieve VEC experiences as the user navigates through multiple screens of the app.
```

Comment:
- please remove the word "single".

- Please remove "It should be noted" and start the sentence with "No network requests...".